src/Gateway.js

```diff
@@ -35,14 +37,15 @@ function RequestGatewayToken() {
 function Gateway() {
   const wallet = useWallet();
   const { publicKey } = wallet;
-  const { gatekeeperNetwork, stage, clusterUrl } = env.prod;
+  const { gatekeeperNetwork, stage, cluster, clusterUrl } = env.prod;
   console.log(`publicKey:${publicKey}`);
   return (
     /* gatekeeperSendsTransaction disabled is the default behavior and the flag can be omitted */
     <GatewayProvider
       wallet={wallet}
       gatekeeperNetwork={gatekeeperNetwork}
       stage={stage}
+      cluster={cluster}
       clusterUrl={clusterUrl}
       gatekeeperSendsTransaction={false}>
```

Comment:
should remove as this is default

Reply:
if you want to use a test network you have to set this to devnet, ortherwise you end up using devnet on the RC, but mainnet on the gatekeeper

Reply:
I meant remove `gatekeeperSendsTransaction`

Reply:
Oh right, sorry. In this case I added it for demonstration purposes with a comment above saying it's optional